repo_structure/repolint.json

```diff
@@ -11,27 +11,27 @@
         "security-file-matches:file-contents": ["error", {"files": ["SECURITY.md"], "content": "https://wiki.hyperledger.org/display/SEC/Defect.Response", "fail-on-non-existent": true}],
         "readme-file-exists:file-existence": ["error", {"files": ["README.md", "README"]}],
         "readme-references-license:file-contents": ["error", {"files": ["README.md", "README"], "content": "license", "flags": "i"}],
-        "maintainers-file-exists:file-existence": ["error", {"files": ["MAINTAINERS.md", "MAINTAINERS.rst"]}],
+        "maintainers-file-exists:file-existence": ["error", {"globsAny": ["MAINTAINERS.rst", "MAINTAINERS.md"]}],
```

Comment:
I have to admit not to understand the difference between files and globsAny. Looking at the repolinter code it seems to have the same result. It might be that globsAny is the new (better?) way but since we use files everywhere else this change looks odd. I would rather change it everywhere if we want to go that way.

Reply:
Files doesn't work as expected.  What this line is meant to check is "at least one of MAINTAIERS.md or MAINTAINERS.rst", files check for both IIRC.

Reply:
Really I think we converged on different solutions, I was working on a regexp version that regressed to a list of names and it looks like you started on that solution.

Reply:
So, do you mind dropping that change so we stay consistent with files throughout?

Reply:
Done